### PR TITLE
Add celonis-ems documentation & related images

### PR DESCRIPTION
**Pull request by capono**

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@capono - Thanks for your submission. Would you take a moment to [sign the CLA](https://github.com/MicrosoftDocs/powerquery-docs/pull/363#issuecomment-1434684927)? After you do so, we can route this PR to the appropriate assignee.

Thanks!

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by capono** [comment]
> @capono - Thanks for your submission. Would you take a moment to [sign the CLA](https://github.com/MicrosoftDocs/powerquery-docs/pull/363#issuecomment-1434684927)? After you do so, we can route this PR to the appropriate assignee.
> 
> Thanks!
> 
> #label:"aq-pr-triaged" @MicrosoftDocs/public-repo-pr-review-team

@ShannonLeavitt  - this PR is due to the Submission of our Connector in ISV-Studio following question came upon agreeing the CLA. 

(i) why we need additional terms for the submission of Documentation when this is already covered under the original Platform Partner Agreement; and (ii) why we’re being asked to agree to the Contribution License Agreement as it does not seem to bear any specific relation to the Connector.

**Comment by Court72** [comment]
@capono,

The Contributor License Agreement (CLA) is an automated check that must be completed during the Validation phase of the pull request process in the public repos. A PR can't be reviewed or merged until all checks are successfully completed.
See [Pull request processing](https://learn.microsoft.com/en-us/contribute/how-to-write-workflows-major#pull-request-processing) for more info.

   image

File: powerquery-docs/connectors/celonis-ems.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Power Query Celonis EMS connector
+description: Provides basic information, prerequisites, and connection steps for Celonis EMS
+author: capono
+
+ms.date: 02/17/2023 
+ms.author:
```
Comment (bezhan-msft) [inline powerquery-docs/connectors/celonis-ems.md:7]: this is 'bezhan'

File: powerquery-docs/connectors/celonis-ems.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+title: Power Query Celonis EMS connector
+description: Provides basic information, prerequisites, and connection steps for Celonis EMS
+author: capono
+
+ms.date: 02/17/2023 
+ms.author: 
+LocalizationGroup: reference
+---
+
+# Celonis EMS
```
Comment (bezhan-msft) [inline powerquery-docs/connectors/celonis-ems.md:11]: This needs to be Celonis EMS (Beta)

File: powerquery-docs/connectors/celonis-ems.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+  
+
+## Summary
+
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
```
Comment (bezhan-msft) [inline powerquery-docs/connectors/celonis-ems.md:19]: Initial release must be Beta

File: powerquery-docs/connectors/celonis-ems.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+| Products | Power BI (Datasets) |
```
Comment (bezhan-msft) [inline powerquery-docs/connectors/celonis-ems.md:20]: This also needs to include Power BI (Dataflows)

File: powerquery-docs/connectors/celonis-ems.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+| Authentication Types Supported | Organizational Account |
+| Function Reference Documentation | [Web.Contents](https://learn.microsoft.com/en-us/powerquery-m/web-contents) </br> [Parquet.Document](xxx) </br> [Json.Document](https://learn.microsoft.com/en-us/powerquery-m/json-document) </br> [Json.FromValue](https://learn.microsoft.com/en-us/powerquery-m/json-fromvalue)|
+
+  
+
+## Prerequisites
+
+Before you can sign in to [Celonis EMS](https://www.celonis.com/ems/platform/?utm_source=google&utm_medium=cpc&utm_campaign=evergreen&utm_content=en_maxconvvalueems_platform_eta&creative=605673291817&keyword=celonis%20ems&matchtype=e&network=g&device=c&_bt=605673291817&_bk=celonis%20ems&_bm=e&_bn=g&_bg=134342462066&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhD5aEPA6iavBUsBG1p0WeGEVi9OF5qcOaohHqNgpbuWa-f9ZHwCtHhoCSqYQAvD_BwE), you must have a Celonis EMS account (username/password).
```
Comment (bezhan-msft) [inline powerquery-docs/connectors/celonis-ems.md:28]: This URL has a lot of parameters... can we simply it?

File: powerquery-docs/connectors/celonis-ems.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (bezhan-msft) [inline powerquery-docs/connectors/celonis-ems.md:1]: This is also missing the changes to the index.md file and the ToC

**Review by bezhan-msft — CHANGES_REQUESTED** [review]

File: powerquery-docs/connectors/celonis-ems.md (hunk 1 above)
Comment (capono) [reply powerquery-docs/connectors/celonis-ems.md]: 'bezhan' has been added as ms.author

File: powerquery-docs/connectors/celonis-ems.md (hunk 2 above)
Comment (capono) [reply powerquery-docs/connectors/celonis-ems.md]: Changed to Celonis EMS (Beta)

File: powerquery-docs/connectors/celonis-ems.md (hunk 3 above)
Comment (capono) [reply powerquery-docs/connectors/celonis-ems.md]: Has also be changed to Beta

File: powerquery-docs/connectors/celonis-ems.md (hunk 4 above)
Comment (capono) [reply powerquery-docs/connectors/celonis-ems.md]: Power BI (Dataflows) was added

File: powerquery-docs/connectors/celonis-ems.md (hunk 5 above)
Comment (capono) [reply powerquery-docs/connectors/celonis-ems.md]: Was simplified.

File: powerquery-docs/connectors/celonis-ems.md (hunk 6 above)
Comment (capono) [reply powerquery-docs/connectors/celonis-ems.md:1]: Currently waiting for the logo - for the index.md file.

File: powerquery-docs/connectors/celonis-ems.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+1. Enter your **Application Key** or your **Personal API Key** generated in the Celonis EMS.
+
+![Screenshot of the authentication dialog where you enter your key for authentication.](media/celonis-ems/authentication.png)
+  
+1. Once you're done, select **Connect**.
+  
+1. Upon a successful connection, the **Navigator** opens with the list of packages available in the given EMS team. Select the **Knowledge Model Record** you want to import and then select **Load**.
+
+![Navigate to package](media/celonis-ems/navigator.png)
+
```
Comment (DougKlopfenstein) [inline powerquery-docs/connectors/celonis-ems.md:50]: Needs to have a separate section for connecting from Power Query Online. You can use the [Azure Blob Storage](https://learn.microsoft.com/power-query/connectors/azure-blob-storage) "Connect to Azure Blob Storage from Power Query Online" section as a template for adding the section to your article.

**Review by DougKlopfenstein — CHANGES_REQUESTED** [review]

**Pull request merged**